pkg/console/subresource/deployment/deployment.go

```diff
@@ -334,3 +337,15 @@ func IsAvailableAndUpdated(deployment *appsv1.Deployment) bool {
 		deployment.Status.ObservedGeneration >= deployment.Generation &&
 		deployment.Status.UpdatedReplicas == deployment.Status.Replicas
 }
+
+func AddVolumeForCustomLogo(operatorConfig *operatorv1.Console) {
+	if logoFile := operatorConfig.Spec.Customization.CustomLogoFile.Name; logoFile != "" {
+		logoPVC := volumeConfig{
```

Comment:
Should this `func` take a `copy` of `volumeConfigList` as an arg so we keep a pristine one around?  Have you tested to ensure this can't append multiple logo file volumes, perhaps if you swap the `logoFile` value several times?

Reply:
Excellent point otherwise the user can never remove the custom logo volume.  I made a working copy and have tested it by passing several different logos in a row and it works well. There can only ever be one mounted logo.

Reply:
Ok great.